components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -108,7 +108,9 @@ constexpr int kHttpUpgradeRequiredStatusResponseCode = 426;
 
 constexpr char kNotificationAdUrlPrefix[] = "https://www.brave.com/ads/?";
 
-BASE_FEATURE(kServing, "AdServing", base::FEATURE_ENABLED_BY_DEFAULT);
+BASE_FEATURE(kServing,
+             "NotificationAdServing",
```

Comment:
we have two features with the same name.

Reply:
"AdServing" was originally in `components/brave_ads/core/internal/ads/serving/serving_features.cc` and `components/brave_ads/browser/ads_service_impl.cc`. Notification ads were decoupled from `serving_features.cc` and both where renamed from `AdServing` to `NotificationAdServing`